gemini/sample-apps/finance-advisor-spanner/pages/1_Asset_Search.py

```diff
@@ -0,0 +1,83 @@
+"""This module is the page for Asset Search feature"""
+
+# pylint: disable=line-too-long,import-error
+
+from database import display_spanner_query, fts_query, like_query
+from home import table_columns_layout_setup
+from itables.streamlit import interactive_table
+import streamlit as st
+
+st.logo(
+    "https://storage.googleapis.com/github-repo/generative-ai/sample-apps/finance-advisor-spanner/images/investments.png"
+)
+
+
+def asset_search_precise() -> None:
+    """This function implements Asset search LIKE Query"""
+
+    query_params = []
+    query_params.append(investment_strategy_pt1.strip())
+    query_params.append(and_or_exclude)
+    query_params.append(investment_strategy_pt2.strip())
+    query_params.append(investment_manager.strip())
+    asset_search_common(query_params, "PRECISE")
```

Comment:
This could probably be moved into the area where it's called. Or the structure could be changed:

```py
query_params = [
    investment_strategy_pt1.strip(),
    and_or_exclude,
    investment_strategy_pt2.strip(),
    investment_manager.strip()
]

Reply:
fixed